### ISSUE TITLE
Extend and build on current structure for SRX-tiers to incentivize further network adoption

### ISSUE THREAD

**Issue by TheEqualizing** [StorXNetwork/StorX-Node#8]
To encourage adoption, the current structure of a tier- layered system with minimum SRX token requirements of a node could be extended to accomodate nodes  that will be deployed later, beyond node #4000.

The current tiered system especially adresses the early adopters. Whenever 4,000 nodes are reached these advantages are taken out of the equation, with quite a steep drop-off, basically completely dissipating this mechanism.

 However with almost 2,500 nodes hosted already in a short period of time, it could benefit the network to also incentivize those who join a bit later, by giving early and mid-term adopters a slight advantage.
 Basically I propose to keep a curve for a longer period of time. 
 
 Since the team didnt announce anything regarding staking node requirements after 4,000 nodes this can easily be extended.
 
Even a slight % for coming onboarding nodes, works incentivizing. This can be done without harm to the tokenomics, and in a fair way. 
 
 It would drive adoption if there also was such a system for nodes

*5,000-10,000
*10,000-15,000
*15,000-20,000  
and so on...

or in any other increments that seems fit
